src/mailadm/bot.py

```diff
@@ -43,119 +43,70 @@ def __init__(self, db: DB, account: deltachat.Account):
 
     @account_hookimpl
     def ac_incoming_message(self, message: deltachat.Message):
-        arguments = message.text.split(" ")
-        logging.info("process_incoming_message: %s", message.text)
-        if not self.is_admin_group_message(message):
+        """This method is called on every incoming message and decides what to do with it."""
+        logging.info("new message from %s: %s" % (message.get_sender_contact().addr, message.text))
+        if self.is_admin_group_message(message):
+            if message.text[0] == "/":
+                logging.info("%s seems to be a valid command.", message.text)
+                self.handle_command(message)
+            else:
+                logging.info("ignoring message, it's just admins discussing in the admin group")
```

Comment:
Hm, we could make a debug log statement out of it? I like to have this else case here for code readability, because it reveals the implicit assumptions I had when I wrote this.